### ISSUE TITLE
this not an isssue just a question.

### ISSUE THREAD

**Issue by vincentroman** [yTakkar/React-Instagram-Clone-2.0#52]
how can i deploy this on a production server like aws or with serverless

**Comment by kxpvcr** [comment]
Have you read [the installation instructions](https://github.com/yTakkar/React-Instagram-Clone-2.0#usage)?

**Comment by StarAzure** [comment]
The instructions do not tell anything about deployment to production. For example, what files/folders have to be uploaded, does it need an index.html etc.

**Comment by StarAzure** [comment]
@vincentroman - If you figured out how to deploy this to production, do you mind sharing?

**Comment by vincentroman** [comment]
yeah actually i have got a way to deploy it to production and for this app it's even a lot easier i will wirte up a tutorial here in few days time sorry about that that

**Comment by StarAzure** [comment]
@vincentroman H Vincent - If you do not have time for a full tutorial write up, a few steps on high level will  get me going and I can figure out. I  am almost ready to move this to prod so any info is appreciated. In return, I have hot reloading working for all files, that way I can share what I have to others. Thanks !

**Comment by vincentroman** [comment]
ok if that's the case you can either upload the app to your production enviroment and run nopm with the --production and use pm2 to manage it as a service and use nginx as reverse proxy and load balancer in orther to use port 80 or 443  and you are good to go

**Comment by StarAzure** [comment]
@vincentroman .. lol .. I will wait for your tutorial .. I don't understand fully.. your comment went right through my head although I know servers ..thanks ..

**Comment by arize99** [comment]
@vincentroman any video or something to explain that broadly? I don't know MERN Stack. I would like to use this script, I know the PHP Version, I am having problems with that also but its PHP I might crack it if I give it more time.

But I need this MERN Version of the script.

**Comment by vincentroman** [comment]
the simplest way to go about hosting this for production without much expertise need is to reapet the same process you did on the dev environment but with few changes like running npm install --production
and changin webpack build to webpack -p --progress --color in the package.json and then install pm2 on the server and use pm2 to run the web app by using pm2 start and that all any other issue just send me a message here i will be glad to help

**Comment by vincentroman** [comment]
dont worry about the aws card issue its just to verify the card plus its just a dollar charge, you can also try the google cloud platform they offer far better option than aws
